src/pages/console/formbuilder/customize.mdx

```diff
@@ -0,0 +1,107 @@
+export const meta = {
+  title: "Customize form inputs",
+  description: "Use the Form Builder in Amplify Studio to customize React form components. You can add new form inputs, bind them to a field, customize labels, and add validation rules."
+}
+
+Use the Form Builder in Amplify Studio to customize React form components. You can add new form inputs, bind them to a field, customize labels, and add validation rules.
+
+![Amplify Studio Form Builder interface breakdown](/images/console/formbuilder/customize-overview.png)
+
+The Form Builder interface consists of the following items:
+1. **Form input**: Select a form input to reveal the **Display**, **Data**, and **Validation rules** settings.
+2. **Display options**: Configure the input **type**, **label**, **placeholder**, and **description**.
+3. **Data**: Connect the input to a field in the submission payload.
+4. **Validation rules**: Visually add validation rules. Learn more about [How to add validation rules](/console/formbuilder/validations/).
+5. **Action menu**: Action menu to delete a form input.
+6. **Form spacing editor**: Set form padding, and horizontal and vertical gaps between inputs.
+7. **View as end user**: Test the form inline, without needing to integrate the form into a React app.
+
+## Add a form input
+
+Add form inputs to personalize the form to your use case. To add a form element:
+1. Move your mouse onto the center form canvas.
+2. Hover in between, before, or after a form input until a blue bar with a (+) sign appears.
+3. Click the blue bar with the (+) sign.
+4. Select the form input you want to add. If your form is connected to a data model, you can also select a field of the data model and let Amplify Studio infer the input.
+
+<video autoPlay={true} muted={true} loop={true} width="100%">
+  <source src="/images/console/formbuilder/customize-add.mp4"/>
+</video>
+
+## Reorder form inputs with drag and drop
+
+Rearrange form inputs vertically or horizontally.
+1. Click and drag a form input
+2. Move form input above, below, left, or right of another form input
+3. Drop when a blue bar appears, indicating the form input's new placement
+
+<video autoPlay={true} muted={true} loop={true} width="100%">
+  <source src="/images/console/formbuilder/customize-drag-and-drop.mp4"/>
+</video>
+
+## View form as end user
+
+If you want to quickly test your form customizations, select **View as end user** to view the customized form fully functioning within Studio.
+
+**Note:** If your form is connected to data model, then form data will be saved to the cloud upon clicking **Submit**. Careful to not accidentally add unintended records into a production database.
```

Comment:
`connected to a data model`

Reply:
`Careful to not accidentally add unintended records into a production database.`

This phrase is a bit weird. Maybe `If this app is being used in production, the data you are saving may be visible to customers.`

Reply:
`Careful to not accidentally add unintended records into a production database.`

This phrase is a bit weird. Maybe `If this app is being used in production, the data you are saving may be visible to customers.`